.github/workflows/e2e-tests-workflow.yml

```diff
@@ -63,15 +64,16 @@ jobs:
           plugin_version=$(node -pe "require('./package.json').version")
           echo odfe version: $odfe_version
           echo plugin version: $plugin_version
+          echo plugin name: $AD_KIBANA_PLUGIN_NAME
           if docker pull opendistroforelasticsearch/opendistroforelasticsearch:$odfe_version
           then
             ## Populate the Dockerfiles
             echo "FROM opendistroforelasticsearch/opendistroforelasticsearch:$odfe_version" >> Dockerfile-AD
             echo "FROM opendistroforelasticsearch/opendistroforelasticsearch-kibana:$odfe_version" >> Dockerfile-AD-Kibana
-            echo "COPY build/opendistro-anomaly-detection-kibana-$plugin_version.zip ." >> Dockerfile-AD-Kibana
+            echo "COPY build/$AD_KIBANA_PLUGIN_NAME-$plugin_version.zip ." >> Dockerfile-AD-Kibana
             ## Uninstall existing AD artifact and install new one
-            echo "RUN if [ -d /usr/share/kibana/plugins/opendistro-anomaly-detection-kibana ]; then /usr/share/kibana/bin/kibana-plugin remove opendistro-anomaly-detection-kibana; fi" >> Dockerfile-AD-Kibana
-            echo "RUN bin/kibana-plugin install file:///usr/share/kibana/opendistro-anomaly-detection-kibana-$plugin_version.zip;" >> Dockerfile-AD-Kibana
+            echo "RUN if [ -d /usr/share/kibana/plugins/opendistro-anomaly-detection-kibana ]; then /usr/share/kibana/bin/kibana-plugin remove $AD_KIBANA_PLUGIN_NAME; fi" >> Dockerfile-AD-Kibana
```

Comment:
Will need to change the first part of this line too - from `opendistro-anomaly-detection-kibana` to `AD_KIBANA_PLUGIN_NAME`

Reply:
good catch. done